src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -818,6 +956,36 @@ private static String readPropertiesContent(int beginLineNo, List<String> lines)
         return stringBuilder.toString();
     }
 
+    private static void validateDefaultProperties(
+        List<String> lines,
+        int beginLineNo,
+        Map<String, String> expectedDefaults) throws IOException, CheckstyleException {
+
+        final String propertyContent = readPropertiesContent(beginLineNo, lines);
+        final Map<Object, Object> properties = loadProperties(propertyContent);
```

Comment:
```suggestion
    private static void validateDefaultProperties(
        Map<Object, Object> properties,
        Map<String, String> expectedDefaults) throws IOException, CheckstyleException {
```
reduce coupling by always giving method what it actually needs to avoid feature envy.

Reply:
done.